RepositoryIT is broken

$ mvn clean install -Pqulice
This command doesn't work.
We should fix it or describe conditions that needed to run the test.


[INFO] Running com.artpie.nuget.RepositoryIT
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.796 s <<< FAILURE! - in com.artpie.nuget.RepositoryIT
[ERROR] com.artpie.nuget.RepositoryIT.shouldInstallAddedPackage  Time elapsed: 0.619 s  <<< ERROR!
java.io.IOException: Cannot run program "nuget" (in directory "/var/folders/q_/xvx_779n3m55fb8xhb6c3lvr0000gq/T/junit4419606334961783779/project"): error=2, No such file or directory
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
	at com.artpie.nuget.RepositoryIT.run(RepositoryIT.java:114)
	at com.artpie.nuget.RepositoryIT.shouldInstallAddedPackage(RepositoryIT.java:69)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:628)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:117)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:184)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:180)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:127)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:135)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)

--- a/README.md
+++ b/README.md
@@ -57,3 +57,11 @@ $ mvn clean install -Pqulice
 ```
 
 To avoid build errors use Maven 3.2+.
+
+The test suite of this project include some integration tests which require NuGet client to be installed.
+NuGet client may be downloaded from official site [nuget.org](https://www.nuget.org/downloads).
+Integration tests could also be skipped using Maven's `skipITs` options:
+
+```
+$ mvn clean install -Pqulice -DskipITs
+```

